docs/source/using/profiles.adoc

```diff
@@ -8,18 +8,45 @@ include::variables.adoc[]
 
 toc::[]
 
-[[pofile-overview]]
+[[profile-overview]]
 == Overview
 
-{project} allows to create multiple instances of {project}.
-Each {project} instance can be created with different configurations and users can switch between different instances.
+The profile feature allows creating multiple instances of {project}. 
 
-[[creating-profile]]
+A profile is an instance of the {project} VM along with all of its configuration and state information.
+
+Each {project} profile can be created with its own configuration (memory, CPU, disk size, add-ons, and so on) and completely independent of other profiles.
+Profiles share nothing between them. Things like cache, add-ons and persistent config are specific to a single profile.
+This means if you have set some specific configuration parameters for a profile, these settings do not apply to other profiles.
+
+Currently, we cannot set any global configuration for profiles.
+If you want to apply a common configuration to all your profiles then you need to apply the settings for each profile separately.
+As a workaround, xref:../using/basic-usage.adoc#environment-variables[environment variables] can be used.
+
+To set 8GB memory for all profiles on Linux/macOS, run:
+
+----
+$ export MINISHIFT_MEMORY=8G
+----
+
+You can have multiple profiles and can switch between them. However only one profile can be active at any time. 
+you do not need to delete an existing {project} instance to test a different configuration.
+
+By default {project} commands will run against the active profile.
```

Comment:
This is kind of repetitive to the sentence above. I guess this can somehow be connected in a better way.